onert-micro/luci-interpreter/src/core/RuntimeModule.h

```diff
@@ -32,9 +32,12 @@ class RuntimeModule
 public:
   RuntimeModule() = default;
 
-  RuntimeGraph *addGraph(IMemoryManager *memory_manager)
+  IBaseRuntimeGraph *addGraph(IMemoryManager *memory_manager)
   {
-    _graphs.push_back(std::make_unique<RuntimeGraph>(this, memory_manager));
+    if (memory_manager->is_static_manager())
```

Comment:
Looking at this code I am thinking maybe it is a good idea to make memory manager internal and do not expose as part of user API.

I mean, just leave it as a boolean option in Interpreter contructor